readme.md

```diff
@@ -1,48 +1,81 @@
-# BEFORE YOU PUBLISH
-- Read [Libraries van Kaliber](https://docs.google.com/document/d/1FrJi-xWtKkbocyMVK5A5_hupjl5E4gD4rDvATDlxWyc/edit#heading=h.bb3md3gyf493).
-- Make sure your example works.
-- Make sure your package.json is correct. Have you change the library title?
-- Update the bin/postInstall script. It should refer to your library.
-- Update the `<title>` tag in `index.html.js`.
-- Remove 'BEFORE YOU PUBLISH' and 'PUBLISHING' from this document.
-
-# PUBLISHING
-- Make sure you are added to the kaliber organization on NPM
-- run `yarn publish`
-- Enter a correct version, we adhere to semantic versioning (semver)
-- run `git push`
-- run `git push --tags`
-- Send everybody an email to introduce them to your library!
-
-# Library title
-Short description.
+# @kaliber/use-media-query
+A hook that makes it easy to use a media query in React.
 
 ## Motivation
-Optionally add a bit of text describing why this library exists.
+Sometimes it's useful to render a different version of a component on different media. For instance: a navigation component can have wildly different implementations for mobile and desktop. This hook gives you the power to use media queries to trigger this behavior.
 
 ## Installation
 
 ```
-yarn add @kaliber/library
+yarn add @kaliber/use-media-query
+```
+
+Add the library to your `compileWithBabel` array:
+
+__`config/default.js`__
+```js
+module.exports = {
+  kaliber: {
+    compileWithBabel: [/@kaliber\/use-media-query/]
+  }
+}
 ```
 
 ## Usage
-Short example. If your library has multiple ways to use it, show the most used one and refer to `/example` for further examples.
 
 ```jsx
-import { hello } from 'library'
+import { useMediaQuery } from '@kaliber/use-media-query'
 
 function Component() {
-  return <div>{hello()}</div>
+  const showDesktopNavigation = useMediaQuery(viewportMd)
```

Comment:
Wat zit er in `viewportMd`?

Reply:
Is het niet slimmer om in de documentatie de letterlijke string te geven, of de variabele bovenin te zetten of een import vanuit .css te doen?